lib/__tests__/standalone-ext.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const path = require('path');
+const standalone = require('../standalone');
+const fixturesPath = path.join(__dirname, 'fixtures');
+
+it('extensions array', async () => {
+	const inputFiles = [
+		path.join(fixturesPath, 'no-empty-source.html'),
+		path.join(fixturesPath, 'extensions'),
+	];
+	const { results } = await standalone({
+		files: inputFiles,
+		customSyntax: 'postcss-scss',
+		extensions: ['css', 'scss'],
+		config: {
+			rules: { 'color-named': 'never' },
+		},
+	});
+
+	expect(results).toHaveLength(3);
+	expect(results[0].source.endsWith('no-empty-source.html')).toBe(true);
+	expect(results[1].source.endsWith(path.join('extensions', 'bar.scss'))).toBe(true);
+	expect(results[2].source.endsWith(path.join('extensions', 'folder', 'qux.css'))).toBe(true);
+});
+
+it('extensions default', async () => {
+	const { results } = await standalone({
+		files: [path.join(fixturesPath, 'extensions')],
+		config: { rules: { 'color-named': 'never' } },
+	});
+
+	expect(results).toHaveLength(1);
+	expect(results[0].source.endsWith(path.join('extensions', 'folder', 'qux.css'))).toBe(true);
+});
```

Comment:
[suggest] Could you add cases when specifying glob patterns?

Reply:
https://github.com/stylelint/stylelint/pull/6617/files#diff-0378929f7797bb789a616f16b2a0f5d0e53a28eba60469829002b0b30d60886cR456

Reply:
https://github.com/stylelint/stylelint/pull/6617/files#diff-0378929f7797bb789a616f16b2a0f5d0e53a28eba60469829002b0b30d60886cR457

Reply:
https://github.com/stylelint/stylelint/pull/6617/files#diff-0378929f7797bb789a616f16b2a0f5d0e53a28eba60469829002b0b30d60886cR457

Reply:
https://github.com/stylelint/stylelint/pull/6617/files#diff-0378929f7797bb789a616f16b2a0f5d0e53a28eba60469829002b0b30d60886cR457

Reply:
I think we need tests in both `cli.test.js` and `standalone.test.js`.
e.g.
- if meow ever let's us detect the case `--ext` no arg
- when meow will support list+multiple

Reply:
`cli.test.js` should focus on handling CLI arguments. We can test the logic details in `standalone-ext.test.js`.

Reply:
So to be clear you want to move all existing tests for `--ext` currently in `cli.test.js` in `standalone-ext.test.js`?

Reply:
If there are cases specific to CLI arguments, keeping them in `cli.test.js` is possible.

But I think we should have cases about extensions filtering in `standalone-ext.test.js` instead of `cli.test.js`.

Reply:
I found a bug while migrating the tests. Ill convert the PR to a draft while trying to fix it.

Reply:
I have just fixed [an edge case](https://github.com/stylelint/stylelint/pull/6617/files#diff-7d9373b505908cd16088030eae195fbcb90036ed455e04eb15b32fffb4feab4aR63) while migrating the tests.
Just to be clear, Iv followed ESLint footsteps: globs are ignored.
Do you have any other cases to cover?